package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "validation-utils",
-  "version": "6.3.0",
+  "version": "7.0.0",
```

Comment:
Why do you consider this a breaking change? This potentially might not compile in real cases?

Reply:
The server requires increasing the major version when changing the existing public API. I think it **can** break compilation in specific cases like when non-nil return type of `validateNil` is used